internal/commands/util/usercount/bitbucket.go

```diff
@@ -0,0 +1,231 @@
+	passwordFlagUsage            = "App password for Bitbucket authentication.Requires read on “Workspace membership“ and “Repositories“ permissions"
+	missingWorkspace             = "Provide at least one workspace"
+	bitbucketManyWorkspaceOnRepo = "You must provide a single workspace for repo counting"
+	bitBucketBot                 = "[bot]"
+)
+
+var (
+	BitBucketWorkspaces []string
+	BitBucketRepos      []string
+	BitBucketURL        *string
+	BitBucketPassword   *string
+	BitBucketUsername   *string
+)
+
+func newUserCountBitBucketCommand(bitBucketWrapper wrappers.BitBucketWrapper) *cobra.Command {
+	userCountCmd := &cobra.Command{
+		Use:     BitBucketCommand,
+		Short:   BitBucketShort,
+		PreRunE: preRunBitBucketUserCount,
+		RunE:    createRunBitBucketUserCountFunc(bitBucketWrapper),
+	}
+
+	BitBucketURL = userCountCmd.Flags().String(urlBitbucket, bitbucketAPIURL, params.URLFlagUsage)
+	userCountCmd.PersistentFlags().StringSliceVar(&BitBucketWorkspaces, workspaceFlag, []string{}, workspaceFlagUsage)
+	userCountCmd.Flags().StringSliceVar(&BitBucketRepos, ReposFlag, []string{}, reposFlagUsage)
+	BitBucketUsername = userCountCmd.Flags().String(params.UsernameFlag, "", usernameFlagUsage)
+	BitBucketPassword = userCountCmd.Flags().String(params.PasswordFlag, "", passwordFlagUsage)
+
+	return userCountCmd
+}
+
+func preRunBitBucketUserCount(*cobra.Command, []string) error {
+	if len(BitBucketWorkspaces) == 0 {
+		return errors.New(missingWorkspace)
+	}
+	if len(BitBucketRepos) > 0 && len(BitBucketWorkspaces) == 0 {
+		return errors.New(missingWorkspace)
+	}
+	if len(BitBucketRepos) > 0 && len(BitBucketWorkspaces) > 1 {
+		return errors.New(bitbucketManyWorkspaceOnRepo)
+	}
+	return nil
+}
+
+func createRunBitBucketUserCountFunc(bitBucketWrapper wrappers.BitBucketWrapper) func(cmd *cobra.Command, args []string) error {
+	return func(cmd *cobra.Command, args []string) error {
+		var totalCommits []wrappers.BitBucketRootCommit
+		var err error
+		var totalContrib uint64 = 0
+		var views []RepositoryView
+		var viewsUsers []UserView
+
+		// In case there is the repos flag
+		if len(BitBucketRepos) > 0 {
+			totalCommits, views, viewsUsers, err = collectFromBitBucketRepos(bitBucketWrapper)
+		} else {
+			totalCommits, views, viewsUsers, err = collectFromBitBucketWorkspace(bitBucketWrapper)
+		}
+		if err != nil {
+			return err
+		}
+		for _, commits := range totalCommits {
+			totalContrib += uint64(len(getUniqueContributorsBitbucket(commits)))
+		}
+
+		views = append(
+			views,
+			RepositoryView{
+				Name:               TotalContributorsName,
+				UniqueContributors: totalContrib,
+			},
+		)
+
+		err = printer.Print(cmd.OutOrStdout(), views, format)
+
+		// Only print user count information if in debug mode
+		if viper.GetBool(params.DebugFlag) {
+			err = printer.Print(cmd.OutOrStdout(), viewsUsers, format)
+		}
+
+		log.Println("Note: dependabot is not counted but other bots might be considered as contributors.")
```

Comment:
Could we set this as a constant in usercount.go so all SCMs get the correct wording?

Reply:
yes, will do